node-launchpad/src/utils.rs

```diff
@@ -135,12 +127,12 @@ macro_rules! trace_dbg {
 
 pub fn version() -> String {
```

Comment:
Looks like you have modified this function, but can we also:
- get rid of `author` from here
- remove protocol version st
- remove data_dir_path
?